File: weatherInfoProj/ClientApp/src/app/Models/weatherInfo.ts
import { DomSanitizer  } from '@angular/platform-browser';

export class WeatherInfo {
    public imageUrl:any;
    constructor(public city: string,public tmp: string,public feelLike: string,
                public wind: string) {

        this.imageUrl = +this.tmp >= 20 ? 'url(assets/Images/sunny.jpeg)':
                                          'url(assets/Images/winter.jpeg)';  
    }

}

File: weatherInfoProj/Models/WeatherInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace weatherInfoProj.Models
{
    public class WeatherInfo
    {
        public string city { get; set; }
        public float tmp { get; set; }
        public float feelLike { get; set; }
        public float wind { get; set; }
    }
}
